src/Elmah.AzureTableStorage/AzureTableStorageErrorLog.cs

```diff
@@ -96,7 +93,8 @@ public override string Log(Error error)
                 StatusCode = error.StatusCode,
                 Type = error.Type,
                 User = error.User,
-            };
+            };
+            elmahEntity.SetXml(ErrorXml.EncodeString(error));
```

Comment:
I agree. Storing the data in two different ways on a continuous basis is confusing.